### PR TITLE
871372: Added UG for Maui SfChat Overview 

### PR DESCRIPTION
**Pull request by RavikumarAkuthotaSf4250**
Added UG for Maui SfChat Overview

### CODE REVIEW

File: MAUI/Chat/overview.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+## Key features
+
+* **Message types:** Incorporated functionality for diverse message formats, including text, calendar, date picker, time picker, and clickable hyperlinks to display a range of message types.
+* **Message input area:** Send messages to users by interacting with the built-in editor and button at the bottom of the chat control.
+* **Time break:** Organizes chat messages into groups according to the timestamp denoted by `System.DateTime` indicating when each message was generated.
+* **Typing indicator:** Utilizes animation to signify users currently engaged in typing activity.
+* **Suggestions:** Includes a selection of options for users to choose from as a response to their message.
+
+![overview in MAUI SfChat](Images/maui-chat-overview.png)
```
Comment (PiruthivirajM) [inline MAUI/Chat/overview.md:22]: do not use "MAUI" word alone.  Say like ".NET MAUI"

File: MAUI/Chat/overview.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+documentation: ug
+---
+
+# Overview of .NET MAUI Chat (SfChat)
+
+The Syncfusion .NET MAUI Chat control offers a contemporary conversational chatbot experience, allowing for flexible customization of the layout to display conversations between two or more users.
+
+## Key features
+
+* **Message types:** Incorporated functionality for diverse message formats, including text, calendar, date picker, time picker, and clickable hyperlinks to display a range of message types.
```
Comment (PiruthivirajM) [inline MAUI/Chat/overview.md:16]: pls reduce the content of "Message Types"

Comment (RavikumarAkuthotaSf4250) [reply MAUI/Chat/overview.md:16]: Changed the content and replaced with Xamarin content

**Review by Lakshminatarajan — APPROVED** [review]
Approved

**Pull request merged**